openc3-cosmos-cmd-tlm-api/spec/controllers/screens_controller_spec.rb

```diff
@@ -103,19 +103,19 @@ def self.find(scope, target, screen)
 
   describe "destroy" do
     it "returns ok" do
-      # Override Screen.all to return a fake list of files
       class Screen < OpenC3::TargetFile
         def self.destroy(scope, target, screen)
+          # Override Screen.destroy to do nothing
```

Comment:
It doesn't like a method without anything ... so I moved all the comments to inside the methods